readme.md

```diff
@@ -1,4 +1,4 @@
-# The DOM (Document Object Model) & JQuery Basics
+# The DOM (Document Object Model) & jQuery Basics
 
```

Comment:
I don't like the `(Document Object Model)` in the title. Makes it look too long.